File: Data Structures & Algorithms/remove-element/submission-0.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        nums.erase(std::remove_if(nums.begin(), nums.end(), [&val](const int& num) {
            return num == val;
        }), nums.end());

        return nums.size();
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cs
public class Solution {
    public int LongestConsecutive(int[] nums) {
        var set = new HashSet<int>();

        foreach (var num in nums) {
            set.Add(num);
        }

        int length = 0;
        foreach (var num in set) {
            if (set.Contains(num - 1)) continue;
            int temp = num;
            int tempLength = 1;

            while (set.Contains(temp + 1)) {
                tempLength++;
                temp++;
            }

            if (tempLength > length) length = tempLength;
        }

        return length;
    }
}
